tasks/copy_to_glacier/test/unit_tests/test_copy_to_glacier.py

```diff
@@ -65,52 +66,70 @@ class TestCopyToGlacierHandler(TestCase):
                         "type": "",
                         "filename": "s3://orca-sandbox-public/MOD09GQ/006/MOD09GQ.A2017025.h21v00.006.2017034065109_ndvi.jpg",
                         "filepath": "MOD09GQ/006/MOD09GQ.A2017025.h21v00.006.2017034065109_ndvi.jpg",
-                        "duplicate_found": True
+                        "duplicate_found": True,
                     },
                     {
                         "name": "MOD09GQ.A2017025.h21v00.006.2017034065109.cmr.xml",
                         "bucket": "orca-sandbox-private",
                         "filename": "s3://orca-sandbox-private/MOD09GQ/006/MOD09GQ.A2017025.h21v00.006.2017034065109.cmr.xml",
                         "type": "metadata",
                         "filepath": "MOD09GQ/006/MOD09GQ.A2017025.h21v00.006.2017034065109.cmr.xml",
-                        "url_path": "MOD09GQ/006"
-                    }
-                ]
+                        "url_path": "MOD09GQ/006",
+                    },
+                ],
             }
         ]
     }
+    # Create the mock for SQS unit tests
+    mock_sqs = mock_sqs()
+
+    def setUp(self):
```

Comment:
Since these apply to every test, these functions should be at the top of the file.

Reply:
I concur the following should be at the top
```
# Create the mock for SQS unit tests
    mock_sqs = mock_sqs()
```

I am wondering if your `SetUp` and `tearDown` should be a part of your SQS test since they are currently not needed by every test and only the one `test_post_to_metadata_queue_happy_path` test currently. Unless we feel this setup/teardown will be needed by a lot more tests after implementation of ORCA-230. Thoughts @AndrewDorn?

Reply:
moved to the top.

maybe keep `SetUp` and `tearDown` for now and we can make the change in orca-230 if needed

Reply:
sounds good

Reply:
https://bugs.earthdata.nasa.gov/browse/ORCA-275